front/public/locales/fr/operationalStudies/scenario.json

```
@@ -40,7 +40,9 @@
     "stopsCount_other": "{{count}} arrêts",
     "trainAdded": "Train ajouté",
     "trainDeleted": "Le train {{name}} a bien été supprimé.",
-    "update": "Modifier"
+    "update": "Modifier",
+    "trainsSelectionDeletedCount_one": "Le train a bien été supprimés.",
```

Oops you forgot the "s" here
```suggestion
    "trainsSelectionDeletedCount_one": "Le train a bien été supprimé.",
```